_layouts/home.html

```diff
@@ -49,6 +48,8 @@ <h1>Newsletter</h1>
       <input type="submit" value="S'inscrire">
     </form>
   </div>
-</div></div>
+<br/>
+<iframe allowfullscreen sandbox="allow-top-navigation allow-scripts allow-popups allow-popups-to-escape-sandbox" width="400" height="800" src="https://mastofeed.com/apiv2/feed?userurl=https%3A%2F%2Fmapstodon.space%2Fusers%2FJourneesQgis&theme=light&size=100&header=true&replies=false&boosts=false"></iframe>
```

Comment:
```suggestion
<iframe allowfullscreen sandbox="allow-top-navigation allow-scripts allow-popups allow-popups-to-escape-sandbox" width="400" height="800" src="https://mastofeed.com/apiv2/feed?userurl=https%3A%2F%2Fmapstodon.space%2Fusers%2FJourneesQgis&theme=light&size=100&header=true&replies=false&boosts=false" style="border-radius: 10px;"></iframe>
```

Histoire d'avoir les bords arrondis comme le formulaire newsletter

Reply:
Ah zut, il y avait l'automerge haha

J'ai ajouté la suggestion et approuvé la PR en même temps haha, je rajoute ça sur master...

Et sinon @haubourg super d'avoir changé le compte pour les Journées QGIS, je n'avais même pas connaissance de ce compte ^^ et super d'avoir uniformisé l'iframe. C'est bien mieux comme ça ;) Merci